_posts/2020-03-28-learning-common-keyboard-shortuts-for-screenreaders.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: "Learning common keyboard shortcuts for screen readers"
+description: "Keyboard shortcuts are hard to learn. Here are some other ways that can help that learning."
+author: anevins12
+published: true
+categories:
+  - Assistive Technology
+further_reading:
+  - url: https://dequeuniversity.com/screenreaders/
+    title: "Screen Reader Keyboard Shortcuts and Gestures"
+    source: Deque University
+  - url: https://gist.github.com/anevins12/a9cc6548fcbab5f55c1dc3c7bbb0f4bd
+    title: "Source JSON file of the mindmap that can be imported into http://app.mindmapmaker.org/"
+    source: Github
+---
+
+Have you ever felt that keyboard shortcuts for assistive technology don't stick in your head? You're not alone, I've been trying to memorise them too. Keyboard shortcuts are often presented to us in a list or a table. Lists and tables are great ways of conveying information, but sometimes a more visual approach could help us learn those shortcuts.
```

Comment:
I’d suggest moving away from writing in the first person. Its atypical with other articles on the site.

Reply:
I think I need to get familiar with those articles

Reply:
I hope I've removed the first person style and I'm happy to learn where this is still an issue